Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM ubuntu:14.04
+# Default arch. Pass in like "--build-arg arch=arm64".
+#   Our circleci arm64 build uses this specifically.
+#   https://docs.docker.com/engine/reference/commandline/build/
+ARG arch=x86_64
+
+# Multiarch Ubuntu Bionic 18.04. arches: x86_64, arm64, etc.
+#   https://hub.docker.com/r/multiarch/ubuntu-core/tags/
+FROM multiarch/ubuntu-core:$arch-bionic
 
 RUN apt-get update && \
```

Comment:
please try if this works without the update, only install? As after updating, much more packages will be rebuilt and it takes much longer.

Reply:
update is needed, some packages are no longer downloadable otherwise. This is a bummber, would have been faster to use just what is in the docker img.